recipes/default.rb

```diff
@@ -34,28 +34,26 @@
 
 if node[:redmine][:ruby_version] == 'system'
   bundle_command = 'bundle'
-  rake_command = 'rake'
-  ruby_command = 'ruby'
+  rake_command = 'bundle exec rake'
+  ruby_command = 'bundle exec ruby'
 else
   # Install ruby with rbenv
-  node.default['rbenv']['user_installs'] = [
-      {
-          user: node[:redmine][:user],
-          rubies: [node[:redmine][:ruby_version]],
-          global: node[:redmine][:ruby_version],
-          gems: {
-              node[:redmine][:ruby_version] => [
-                  { name: 'bundler' }
-              ]
-          }
-      }
+  include_recipe 'rbenv::default'
+  include_recipe 'rbenv::ruby_build'
+
+  node.default['rbenv'][:group_users] = [
+    node[:redmine][:user]
   ]
-  include_recipe 'ruby_build'
-  include_recipe 'rbenv::user'
 
-  bundle_command = "#{node[:redmine][:home]}/.rbenv/shims/bundle"
-  rake_command = "#{node[:redmine][:home]}/.rbenv/shims/rake"
-  ruby_command = "#{node[:redmine][:home]}/.rbenv/shims/ruby"
+  rbenv_ruby(node[:redmine][:ruby_version]) { global true }
+
+  rbenv_gem 'bundler' do
+    ruby_version node[:redmine][:ruby_version]
+  end
+
+  bundle_command = "/opt/rbenv/shims/bundle"
+  rake_command = "#{bundle_command} exec /opt/rbenv/shims/rake"
+  ruby_command = "#{bundle_command} exec /opt/rbenv/shims/ruby"
```

Comment:
Hm, just wondering whether it would make sense to reuse the `ruby-cookbook` here since it sets up most of this stuff already.
Pros:
* already exists and works and we'll reuse stuff
* does some additional thinks like pinning the bundle / rbenv version, etc
Cons:
* might deviate from the setup here
* might not do all of the setup being done here

So, I guess it's a no-go for now but thought I'd raise it to briefly discuss it.